code/datums/martial/baby.dm

```diff
@@ -0,0 +1,48 @@
+/datum/martial_art/baby
+	name = "Baby"
+	id = MARTIALART_BABY
+
+/datum/martial_art/baby/disarm_act(mob/living/carbon/human/Attacker, mob/living/carbon/human/Defender)
+	if(Attacker == Defender)
+		to_chat(Attacker, "<span class='warning'>You can't disarm yourself!</span>")
+		return TRUE
+	if(Attacker != Defender)
+		return FALSE
+	return
+
```

Comment:
This never runs

Reply:
what never runs?

Reply:
The last return.

Reply:
ohhh, i see that now, its been awhile since ive used if statements, i forgot you dont need to define what happens if the if check fails unless you are checking for multiple scenarios, derp